parley/src/layout/line/greedy.rs

```diff
@@ -8,10 +8,13 @@ use swash::text::cluster::Whitespace;
 
 #[cfg(feature = "libm")]
 #[allow(unused_imports)]
-use core_maths::*;
+use core_maths::CoreFloat;
```

Comment:
`core_maths`' docs recommend using the previous style of import. I'm not sure why, as it does only export one thing

Reply:
Yeah doesn't make sense to me, AFAIK the only semantic difference would be that a wildcard import doesn't clash with a `CoreFloat` trait in the current module namespace (which doesn't exist...).

Reply:
I've seen other things do `use core_maths::CoreFloat as _;` or similar for their traits ...